src/constants.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""This module defines constants used throughout the Synapse application."""
+
+CHECK_ALIVE_NAME = "synapse-alive"
+CHECK_MJOLNIR_READY_NAME = "synapse-mjolnir-ready"
+CHECK_NGINX_READY_NAME = "synapse-nginx-ready"
+CHECK_READY_NAME = "synapse-ready"
+COMMAND_MIGRATE_CONFIG = "migrate_config"
+SYNAPSE_CONFIG_DIR = "/data"
+MJOLNIR_CONFIG_PATH = f"{SYNAPSE_CONFIG_DIR}/config/production.yaml"
+MJOLNIR_HEALTH_PORT = 7777
+MJOLNIR_MANAGEMENT_ROOM = "management"
+MJOLNIR_MEMBERSHIP_ROOM = "moderators"
+MJOLNIR_SERVICE_NAME = "mjolnir"
+PEER_RELATION_NAME = "synapse-peers"
+PROMETHEUS_TARGET_PORT = "9000"
+# Disabling it since these are not hardcoded password
+SECRET_ID = "secret-id"  # nosec
```

Comment:
Some of these also seem to be only used in mjoinir.py

Reply:
Related: #52